app/views/admin/registrations/index.pdf.prawn

```diff
@@ -13,10 +12,6 @@ prawn_document(:force_download=>true, :filename => @pdf_filename) do |pdf|
     row << ""
     row << registration.name
     row << registration.email
-    if registration.attending_social_events
-      row << "X"
-    else
-      row << " "
     end
 
     if registration.attending_with_partner.to_s
```

Comment:
I suppose we also moved attending_with_partner!
https://github.com/openSUSE/osem/blob/master/db/schema.rb#L310